extensions/hdrhistogram/src/main/java/org/radargun/stats/HistogramOperationStats.java

```diff
@@ -33,24 +23,15 @@
    @Property(doc = "Number of significant digits. Default is 2.")
    private int digits = 2;
 
-   private AbstractHistogram histogram;
-   private SoftReference<AbstractHistogram> soft;
-   private long errors = 0;
-   private Histogram compacted;
+   private Timer timer;
```

Comment:
Why are you replacing this with another instance rather than embedding the three histograms directly?
I suppose this is one of the 'gaps' you mentioned.

Reply:
Yes. It is a gap yet.